charts/redpanda/Chart.yaml

```
@@ -23,7 +23,7 @@ type: application
 # The chart version and the app version are not the same and will not track
 # together. The chart version is a semver representation of changes to this
 # chart.
-version: 5.3.1
+version: 5.3.3
```

Why bump is +2? Should we have
```suggestion
version: 5.3.2
```

---

Yeah! but I am waiting for another PR to go in first ;)